File: dongyimaimangerweb/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.selectOptionList = function () {
        return $http.get('../brand/selectOptionList.do');
    }

    this.findAll = function () {
        return $http.get('../brand/findAll.do');
    }

    this.findPage = function (page,rows) {
        return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
    }

    this.save = function (methodType,entity) {
        return $http.post('../brand/'+methodType+'.do',entity);
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }

    this.dele = function (ids) {
        return $http.get('../brand/dele.do?ids='+ids);
    }

    this.search = function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
    }

});

File: dongyimaisellergoodsinterface/src/main/java/com/dongyimai/sellergoods/service/BrandService.java
package com.dongyimai.sellergoods.service;

import com.dongyimai.entity.PageResult;
import com.dongyimai.pojo.TbBrand;

import java.util.List;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum, int pageSize);

    public void save(TbBrand brand);

    public TbBrand findOne(Long id);

    public void update(TbBrand brand);

    public void dele(Long[] ids);

    public PageResult search(int page, int rows, TbBrand brand);

    public List<TbBrand> selectOptionList();

}
